scripts/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/ash
+
+printf "\n\n======================================\n"
+printf "Run database migrations"
+printf "\n======================================\n\n"
+yarn migrations
+
+printf "\n\n======================================\n"
+printf "Start the application"
+printf "\n======================================\n\n"
+node ./build/index.js
+
+exit 0
```

Comment:
remove the exit 0; by default `node ./build/index.js` will continuously run without reaching the `exit 0` statement

Reply:
Done